### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JamieSlome**
Adds the latest PyPi version badge to the `README.md` 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
     <a href="./LICENSE">
         <img src="https://img.shields.io/github/license/citi/scaler?label=license&colorA=0f1632&colorB=255be3">
     </a>
+    <img alt="PyPI - Version" src="https://img.shields.io/pypi/v/scaler?colorA=0f1632&colorB=255be3">
```
Comment (RaphaelJ) [inline README.md:16]: Could we make it clickable so that it links to https://pypi.org/project/scaler/ ?

**Comment by RaphaelJ** [comment]
Great. I'll make the same for Pargraph and Parfun with the next PR.

File: README.md (hunk 1 above)
Comment (rafa-be) [inline README.md:16]: Could we make it clickable so that it links to https://pypi.org/project/scaler/ ?

**Comment by rafa-be** [comment]
Great. I'll make the same for Pargraph and Parfun with the next PR.

File: README.md (hunk 1 above)
Comment (JamieSlome) [reply README.md]: Done 👍

**Pull request merged**